pages/content/posts/2022-q3-report/index.md

```diff
@@ -0,0 +1,178 @@
+---
+author: "Open Web Docs staff"
+title: "Open Web Docs 2022 Q3 report"
+date: "2022-10-27"
+ShowToc: true
+ShowBreadCrumbs: false
+excerpt: "An update on our work in the third quarter of 2022."
+---
+
+<div class="callout">
+
+In 2022 Q3, OWD:
+
+- was again the main organization maintaining the [mdn/content](https://github.com/mdn/content) repository:
+  - made 1,945 reviews for PRs merged into mdn/content, 63% of all reviews.
+  - made 284.5 of the PRs made to mdn/content, 11% of the total and twice as many as any other organization.
+- led projects to:
+  - update thousands of JS code samples in MDN to use modern practices.
+  - integrate markdownlint into the mdn/content repository.
+  - ensure MDN content and examples follow best practices.
+- launched a website at [https://openwebdocs.org](https://openwebdocs.org) and Carle, the book worm, the new OWD logo.
+
+</div>
+
+This report provides an update on Open Web Docs for Q3 2022. It’s split into the following parts:
+
+1. [**Goals**](#goals): a restatement of the overall goals of the organization.
+2. [**Impact**](#impact): a summary of day to day contributions to documentation projects.
+3. [**Projects**](#projects): a summary of completed projects.
+
+# Goals
+
+Just like in 2021, the main focus of Open Web Docs in 2022 has been contributing to MDN Web Docs.
+
+This takes the form of contributions to the main content repositories under the [https://github.com/mdn](https://github.com/mdn) GitHub organization:
+
+- [mdn/content](https://github.com/mdn) : the main content repository for MDN Web Docs.
+- [mdn/browser-compat-data](https://github.com/mdn) : the source of browser compatibility data.
+- [mdn/learning-area](https://github.com/mdn) : examples supporting the MDN Learning Area.
+- [mdn/interactive-examples](https://github.com/mdn) : source for MDN’s interactive examples.
+
+# Impact
+
+## The value of MDN
+
+MDN is an essential resource for web developers. It:
+
+- [has been cited from Stack Overflow more than 150,000 times in the last 5 years](https://meta.stackoverflow.com/questions/417189/which-sites-are-currently-linked-to-most-often-in-stack-overflow/417190#417190), #2 among all developer documentation sites.
+- was ranked #2 (behind Stack Overflow) in the [State of JS 2021](https://2021.stateofjs.com/en-US/resources/#sites_courses), and [State of CSS 2021](https://2021.stateofcss.com/en-US/resources/#sites_courses) surveys.
+- receives tens of millions of unique visitors every month.
+
+It is therefore essential for the web developer community that MDN continues to be maintained and improved, and the core team continues to be supported in reviewing PRs from the contributor community.
+
+## The contribution of Open Web Docs
```

Comment:
```suggestion
## The contribution of Open Web Docs
```

## The contributions of Open Web Docs

Reply:
Plural for contributions?

Reply:
I think "contribution" is fine here. A collection of "individual things you do to help a project within a given time frame"  can be called "your contribution to the project in that timeframe", although you can also refer to the individual things as contributions. But again I don't really mind either way.